jetbrains-gateway/src/software/aws/toolkits/jetbrains/gateway/CawsConnectionProvider.kt

```diff
@@ -103,10 +107,30 @@ class CawsConnectionProvider : GatewayConnectionProvider {
             return null
         }
 
+        val currentConnection = ToolkitConnectionManager.getInstance(null).activeConnectionForFeature(CodeCatalystConnection.getInstance())
+            as AwsBearerTokenConnection?
```

Comment:
should never be null

Reply:
This can be null in case the user is not signed in within GW. If this is null, there is a step below that asks the users to sign in